README.md

```diff
@@ -39,27 +56,19 @@ Go to `http://127.0.0.1:4567/`
 
 Observe output in `notion.taskpaper`, `notion.markdown`, and `notion.html`.
 
-#### To run the command-line app with dynamic config options
-
-Set `CONFIG` in `config.rb` to `nil`, then re-run the command-line app.
-
-```rb
-CONFIG = nil
-```
-
-It will prompt you for which database, which property to filter by, and which option of that property to filter by.
+#### To run the app with dynamic config options
 
-Remember that this `config.rb` will break the web app.
+Set `CONFIG` in `config.rb` to `nil` or comment it out completely, then re-run the (command-line or web) app. It will prompt you for which database, which property to filter by, and which option of that property to filter by.
```

Comment:
We might want to let the user know `run.rb` also asks them if they want to use the values in `config.rb`.